core/src/main/scala/spinal/core/formal/SymbiYosysBackend.scala

```diff
@@ -29,8 +29,8 @@ sealed trait SbyEngine extends FormalEngin {
 object SmtBmcSolver extends Enumeration {
   type SmtBmcSolver = Value
 
-  val Yices = Value("yices")
-  val Boolector = Value("booleactor")
+  val Yices = Value("yices")      // Yices2 ?
+  val Boolector = Value("booleactor")   // TYPO: boolector
```

Comment:
This would be a TYPO, and I did not add testcase for this one. What have been tested are only yices, z3. It would be better to add all support cases. However, I do not use others. :)

Furthermore, I found that I did not even add boolector into the docker image. :(